.gitignore

```diff
@@ -135,3 +135,7 @@ package-lock.json
 repos_test/
 
 .direnv
+
+# idea
+.idea/
```

Comment:
Can we split this into a separate PR, as it's unrelated to the rest of the changes?